src/components/Heading/Heading.tsx

```diff
@@ -6,14 +6,28 @@ import {
 import React, { forwardRef } from "react";
 
 import Link from "../Link/Link";
+import useNYPLBreakpoints from "../../hooks/useNYPLBreakpoints";
 
 export const headingSizesArray = [
+  "display1",
+  "heading1",
+  "heading2",
+  "heading3",
+  "heading4",
+  "heading5",
+  "heading6",
   "primary",
   "secondary",
   "tertiary",
   "callout",
 ] as const;
 export const headingLevelsArray = [
+  "h1",
+  "h2",
```

Comment:
on line 99, the default should be "h2".

Reply:
I opted to not do that yet because we don't want this update to make anything change visually when updating from a previous version of DS.  That change can be made when we update the components to implement the new styles.

Reply:
True. Can you add a `// TODO: ....` comment as a reminder?